data/json/npcs/robofac/intercom/robofac_intercom_hdd_trade.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "//": "These are all dialogues related to the breadcrumb quest from the refugee center to deliver a hard drive.",
+    "id": [
+      "TALK_ROBOFAC_INTERCOM_FREE_MERCHANT_DELIVERY_1",
+      "TALK_ROBOFAC_INTERCOM_FREE_MERCHANT_DELIVERY_2",
+      "TALK_ROBOFAC_INTERCOM_FREE_MERCHANT_DELIVERY_3"
+    ],
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_has_var": "u_started_robofac_hdd_trade",
+      "type": "dialogue",
+      "context": "intercom",
+      "value": "yes",
+      "yes": "&The intercom's tray wordlessly slides open.",
+      "no": "&The intercom barks as a voice clears its throat, then speaks slowly and carefully.  \"Understood.  There is a tray embedded beneath the intercom.  Please place the drive into it.\"  On cue, a small portion of the wall beneath the intercom slides out along rails, revealing a large tray several inches deep.  \"You are welcome to leave afterwards.\""
```

Comment:
Hmm I wouldn't describe the intercom using biological terms. I know its a pretty small detail, but this is potentially the first interaction a new player get with the faction, so i think its important here. I'd suggest something like:

"An embed speaker erupts into static before resolving to a slow speaking voice"
Although there are other lines like that already hmm.